gluoncv/data/otb/tracking.py

```diff
@@ -0,0 +1,212 @@
+"""Visual Tracker Benchmark."""
+import json
+import os
+import cv2
+from tqdm import tqdm
+
+class Dataset(object):
```

Comment:
Usually we use `dataset.Dataset` from `from mxnet.gluon.data import dataset`. Is there a reason why you write your own Dataset class?

Reply:
I check the ` from mxnet.gluon.data import dataset` and it appears NotImplementedError if `len(dataset)`. I check the `mxnet/gluon/data/dataset`  find `def __len__(self): raise NotImplementedError`.